mission_control/navigator_launch/launch/subsystems/cameras/calibration/00b09d0100e84a42.yaml

```diff
@@ -0,0 +1,20 @@
+image_width: 960
+image_height: 600
+camera_name: 00b09d0100e84a42
+camera_matrix:
+  rows: 3
+  cols: 3
+  data: [702.1922, 0, 476.5935, 0, 699.2620, 298.8085, 0, 0, 1]
+distortion_model: plumb_bob
+distortion_coefficients:
+  rows: 1
+  cols: 5
+  data: [-0.1701, 0.1220, -.00085339, -0.0012, -0.0304]
+rectification_matrix:
+  rows: 3
+  cols: 3
+  data: [0.9996475214662156, 0.003230347798860112, 0.02635142651832315, -0.003259850357149076, 0.9999941070295705, 0.001076699484903968, -0.02634779311633482, -0.001162221678595386, 0.9996521610233574]
+projection_matrix:
+  rows: 3
+  cols: 4
+  data: [702.1922, 0, 476.5935, -40.00, 0, 699.2620, 298.8085, 0, 0, 0, 1, 0]
```

Comment:
@rleyva, the opencv calibration that I did about a month ago, which had the camera intrinsics corroborated by the matlab calibration tool and are really close to the intrinsics you have here, and which also correctly backed out the distance between the cameras and almost identical orientations,  say that the fourth coefficient here should be -140, not -40. 
Look at the [calibration file](https://github.com/uf-mil/Navigator/blob/dev/gnc/navigator_perception/calibration.xml). It has been on Navigator for a while now (although not on the repo) and Tess and I have been using it successfully for stereo stuff.